docs/internal/Level-1+-Reviews-and-Changes-Requested.md

```diff
@@ -0,0 +1,131 @@
+# Level 1+ Reviews and Changes Requested
+
+The [Level 1 Reviews](Level-1-Review-And-LOQ.md) explains how reviews with only 1 level works. Which is a subset of how a review with more levels (per stage) should work.
+
+The aim of this page is to outline the structure changes and flows:
+
+## Consolidation in agreement
+
+- Reviewer (Level 1) - reviewing an application
+- Reviewer (Level 1) - submit ~~`LOQ (List of Questions)`~~ `Conform` or `Non Conform` advise to upper level reviewer
```

Comment:
```suggestion
- Reviewer (Level 1) - submit ~~`LOQ (List of Questions)`~~ `Conform` or `Non Conform` to advise higher level reviewer
```
Also -- is the ~~strikethrough~~ helpful?

Reply:
Yea,no. I'll remove the LOQ part